ios/StripeTerminalReactNative.swift

```diff
@@ -513,6 +525,18 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         sendEvent(withName: ReactNativeConstants.CHANGE_CONNECTION_STATUS.rawValue, body: ["result": result])
     }
 
+    func terminal(_ terminal: Terminal, didStartReaderReconnect cancelable: Cancelable) {
+        sendEvent(withName: ReactNativeConstants.START_READER_RECONNECT.rawValue, body: nil)
+    }
+
+    func terminalDidSucceedReaderReconnect(_ terminal: Terminal) {
+        sendEvent(withName: ReactNativeConstants.TERMINAL_SUCCEED_READER_RECONNECT.rawValue, body: nil)
+    }
```

Comment:
in the Android listener we use mapFromReader to send the reader in body, should that here as well instead of sending `nil`